webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/AsyncComplianceService.scala

```diff
@@ -99,13 +100,15 @@ class AsyncComplianceService(
 
     // Compute compliance level for all rules in  a future so it will be displayed asynchronously
     val futureCompliance: Future[Box[Map[Kind, Option[ComplianceLevel]]]] = {
+      implicit val qc: QueryContext = CurrentUser.queryContext
+
```

Comment:
I wonder if there is something special with `Future` and query context. I don't think it matters in our usage (the user must be logged to call that), but something that may be relevant in complicated bugs at some point.